downsample_cells.R

```diff
@@ -96,14 +106,40 @@ for (field in c(opt$cell_id_field, opt$cell_type_field)){
     quit(status = 1)
   }
 }
+
+metadata[[opt$cell_id_field]] <- gsub(' ', '_', metadata[[opt$cell_id_field]])
```

Comment:
this might cause a mismatch to what we have in the barcodes file, doesn't it?

Reply:
Actually this was positively required to make your test case work. But spaces aren't allowed in the cell identifiers, and will be replaced where present during quantification, so this is probably fairly safe.